### ISSUE TITLE
Contents Cache failed for <Object name>

### ISSUE THREAD

**Issue by TehomCD** [evennia/evennia#1169]
#### Brief summary of issue / Description of requested feature:
This happened before in issue #1064 and I thought it was resolved, but when checking my logs for the past week, I occasionally will see dozens of lines of error messages of `Contents cache failed for <object name here>`, a character or a room. Reloading fixes it, though I'm unsure why this would suddenly pop back after seeming to be resolved for months.

#### Steps to reproduce the issue / Reasons for adding feature:

1. Long enough period between reloads seems to occasionally create this error.

#### Error output / Expected result of feature
Dozens of lines of `Contents cache failed for <object name>`. Usually more than one object is affected - somewhere between 1-6 characters or rooms seems to be typical.

#### Extra information, such as Evennia revision/repo/branch, operating system and ideas for how to solve / implement:
Probably related to issue #1064

**Comment by Griatch** [comment]
In #1064, the reason for this error was the firing of the cache flush mechanism (see my comment that reproduced it). This will start firing if the memory footprint of your game is increasing past the memory limiter breakoff point as set in settings. Is the memory limiter firing for you in the same way now?

**Comment by TehomCD** [comment]
I haven't been able to reproduce this with `@server/flushmem` yet. I'll keep an eye on things and see if I can figure out any way to reliably reproduce this.

**Comment by Griatch** [comment]
When it happens, keep an eye on the memory or CPU consumption (with, say `top` or better `htop`), see if there is some particular thing happening with that.

**Comment by Griatch** [comment]
@TehomCD Is this something that still occurs?

**Comment by TehomCD** [comment]
The last incident of this was about two months ago, so not in quite a while. Since I wasn't keeping a watch on memory usage, it's entirely possible that it was related to the portal using too much memory in some way. It may be resolved if you want to mark it closed, and I could just reopen it if it still happens after we're using devel.

**Comment by Griatch** [comment]
@TehomCD I'm closing this for now, let me know if the issue emerges with Evennia 0.7.

**Comment by aceamro** [comment]
This problem occurs frequently.
It was the same in version 0.7, and the same in version 0.9.
Usually occurs after a few hours.
No signs were found in memory or CPU.
reloading should work fine.

**Comment by Griatch** [comment]
@aceamro: Under what conditions is it occurring? Is it on an active game with lots of players, just an unused game? Lots of Scripts? What?

**Comment by aceamro** [comment]
I'll give you my game source, can I open it and try it?
It's difficult because it's in Korean, but I think it's good to see the situation.
Open it and tell us your address. I'll try to connect with several characters.
Is it possible?

This link is my world source.
https://www.dropbox.com/s/sb49hdce18ispy7/groozam.zip?dl=0

**Comment by Griatch** [comment]
@aceamro Thanks for the source. I set up a server with your code and has had it running for two days with one character logged in (no, it's not a public server so you cannot connect to it from the outside). I did not see any error in the log in this time. 
So since you apparently see this 'after a few hours' then it's either something specific you do in the game that causes it, or it's something strange with your hardware setup.

**Comment by aceamro** [comment]
You worked hard.
One character never had problems,
More than 10 characters use triggers such as tintin,
It happened when a few hours of quick battle.
This problem has been difficult for over a year.
I will keep trying to find the cause.

**Comment by Griatch** [comment]
@aceamro That information (the conditions under which the cache fails) was what I was asking you about from the onset :) 
You also need to tell me for _which objects_ (or type of objects) the cache is failing (it will say in the log) and what exactly those objects are doing at the time ("few hours of quick battle" doesn't tell me much, and I can't read the Korean in the video). It's a too little information for reproducing, unfortunately.

**Comment by aceamro** [comment]
@Griatch Sorry for the lack of explanation.
All types logged were character.
"Contents cache failed for <character name>"

**Comment by aceamro** [comment]
@Griatch 
Once I found something strange

Use the return order to go somewhere,
After hunting and dying back

evennia / objects / models.py
def get (self, exclude = None):

The top part keeps calling every second
I tried to log.

19-10-25 03: 43: 48 + 09 try = [north, west, east, south, skeleton]
19-10-25 03: 43: 48 + 09 if = [1962, 1994, 1997, 1999]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south]
19-10-25 03: 43: 48 + 09 if = [2008, 2040, 2043, 2045, 46436, 46488]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, ghoul, ghoul]
19-10-25 03: 43: 48 + 09 if = [1190, 1222, 1225, 1227, 46253, 46299]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, skeleton, skeleton]
19-10-25 03: 43: 48 + 09 if = [1958, 1990, 1993, 1995, 46502]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, ghoul]
19-10-25 03: 43: 48 + 09 if = [1300, 1332, 1335, 1337]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south]
19-10-25 03: 43: 48 + 09 if = [1194, 1226, 1229, 1231, 46268, 46271]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, skeleton, skeleton]
19-10-25 03: 43: 48 + 09 if = [1186, 1218, 1221, 1223, 46242]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, skeleton]
19-10-25 03: 43: 48 + 09 if = [1262, 1294, 1297, 1299]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south]
19-10-25 03: 43: 48 + 09 if = [1600, 1635, 1637, 46388]
19-10-25 03: 43: 48 + 09 try = [north, east, south, mummy]
19-10-25 03: 43: 48 + 09 if = [1232, 1264, 1267, 1269]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south]
19-10-25 03: 43: 48 + 09 if = [1258, 1290, 1293, 1295, 46211, 46267, 46221]
19-10-25 03: 43: 48 + 09 try = [North, West, East, South, Skeleton, Skeleton, Skeleton]
19-10-25 03: 43: 48 + 09 if = [1254, 1289, 1291, 46294, 46204]
19-10-25 03: 43: 48 + 09 try = [north, east, south, skeleton, skeleton]
19-10-25 03: 43: 48 + 09 if = [1258, 1290, 1293, 1295, 46205, 46211, 46267]
19-10-25 03: 43: 48 + 09 try = [North, West, East, South, Skeleton, Skeleton, Skeleton]
19-10-25 03: 43: 48 + 09 if = [1498, 1530, 1533, 1535]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south]
19-10-25 03: 43: 48 + 09 if = [1190, 1222, 1225, 1227, 46253, 46272]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, skeleton, skeleton]
19-10-25 03: 43: 48 + 09 if = [1296, 1328, 1331, 1333]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south]
19-10-25 03: 43: 48 + 09 if = [1338, 1368, 1369]
19-10-25 03: 43: 48 + 09 try = [north, west, east]
19-10-25 03: 43: 48 + 09 if = [1612, 1644, 1647, 1649]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south]
19-10-25 03: 43: 48 + 09 if = [1194, 1226, 1229, 1231, 46234, 46271]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, skeleton, skeleton]
19-10-25 03: 43: 48 + 09 if = [2004, 2036, 2039, 2041, 46502]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, ghoul]
19-10-25 03: 43: 48 + 09 if = [2008, 2040, 2043, 2045, 46488, 46501]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, ghoul, ghoul]
19-10-25 03: 43: 48 + 09 if = [2008, 2040, 2043, 2045, 46436, 46501]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, ghoul, ghoul]
19-10-25 03: 43: 48 + 09 if = [1054, 1086, 1089, 1091, 46239]
19-10-25 03: 43: 48 + 09 try = [north, west, east, south, skeleton]

**Comment by aceamro** [comment]
@Griatch 
I found a way to test it easily.
If you go anywhere with the @tel command, a similar phenomenon occurs.

**Comment by Griatch** [comment]
@aceamro Interesting. I don't see it with a simple `@tel` between rooms, but it looks like you additionally have a lot of deleted objects (probably because they were killed in combat?). I'll keep testing to see if I can reproduce.

**Comment by aceamro** [comment]
@Griatch After further testing, it was not caused by the move.
It seems to be caused by the triggered object.
I'm sorry.
I don't know if I can prevent the contents cache failed, but I'm running server / flushmem every hour.
I'll keep looking for the cause.

**Comment by aceamro** [comment]
@Griatch I just ran into a problem with my developer account and found it:

@py self.location.contents_cache._idcache
Found too much in

When I did @py self.location.contents_cache.clear () nothing happened,

@py self.location.contents_cache._idcache
Running it again produced the same amount of data.

**1. @py self.location.contents_cache._pkcache**
{47704: None, 54929: None, 54930: None, 54946: None, 56108: None, 56113: None, 57317: None, 60998: None, 62691: None, 62831: None, 63657: None, 69217: None, 69218: None, 76052: None, 76123: None, 77556: None, 96950: None, 96995: None, 97017: None, 56724: None}

**2. @py self.location.contents_cache._idcache**
{56722: 루시퍼, 47732: 사비, 47733: 틴틴, 47734: 제제, 47735: 트리, 47736: 마법사, 56724: 미카엘, 96991: 동네큰형, 96995: 동네대장, 96996: 동네엉아, 96997: 동네형아, 96998: 동네히야, 1: 암로, 46073: 슬라임, 46046: 슬라임, 46102: 슬라임, 46010: 슬라임, 46086: 슬라임, 46124: 좀비, 46113: 좀비, 46248: 스켈레톤, 46216: 스켈레톤, 46275: 스켈레톤, 46282: 스켈레톤, 46288: 스켈레톤, 46371: 머미, 46388: 머미, 46445: 구울, 46473: 구울, 46553: 듀라한, 46524: 듀라한, 46508: 듀라한, 46523: 듀라한, 46008: 슬라임, 46048: 슬라임, 46503: 듀라한, 46529: 듀라한, 46517: 듀라한, 46532: 듀라한, 46539: 듀라한, 46552: 듀라한, 46510: 듀라한, 46562: 듀라한, 46522: 듀라한, 46519: 듀라한, 46527: 듀라한, 46551: 듀라한, 46544: 듀라한, 46520: 듀라한, 46556: 듀라한, 46513: 듀라한, 46518: 듀라한, 46561: 듀라한, 46511: 듀라한, 46538: 듀라한, 46550: 듀라한, 46505: 듀라한, 46271: 스켈레톤, 46292: 스켈레톤, 46573: 듀라한, 46007: 슬라임, 46057: 슬라임, 46054: 슬라임, 46541: 듀라한, 46597: 듀라한, 46549: 듀라한, 46514: 듀라한, 46521: 듀라한, 46528: 듀라한, 46580: 듀라한, 46567: 듀라한, 46581: 듀라한, 46602: 듀라한, 46456: 구울, 46459: 구울, 46526: 듀라한, 46578: 듀라한, 46548: 듀라한, 46595: 듀라한, 46509: 듀라한, 46591: 듀라한, 46486: 구울, 46071: 슬라임, 46099: 슬라임, 46570: 듀라한, 46600: 듀라한, 46576: 듀라한, 46571: 듀라한, 46572: 듀라한, 46587: 듀라한, 46592: 듀라한, 46563: 듀라한, 46565: 듀라한, 46596: 듀라한, 46582: 듀라한, 46593: 듀라한, 46546: 듀라한, 46416: 구울, 46454: 구울, 46601: 듀라한, 46060: 슬라임, 46409: 구울, 46249: 스켈레톤, 46261: 스켈레톤, 46051: 슬라임, 46206: 스켈레톤, 46276: 스켈레톤, 46289: 스켈레톤, 46480: 구울, 46212: 스켈레톤, 46012: 슬라임, 46243: 스켈레톤, 46267: 스켈레톤, 46095: 슬라임, 46021: 슬라임, 46024: 슬라임, 46044: 슬라임, 46100: 슬라임, 46214: 스켈레톤, 46090: 슬라임, 46247: 스켈레톤, 46067: 슬라임, 46093: 슬라임, 46300: 스켈레톤, 46286: 스켈레톤, 46011: 슬라임, 46262: 스켈레톤, 46035: 슬라임, 46058: 슬라임, 46294: 스켈레톤, 46031: 슬라임, 46089: 슬라임, 46065: 슬라임, 46050: 슬라임, 46009: 슬라임, 46052: 슬라임, 46032: 슬라임, 46004: 슬라임, 46019: 슬라임, 46027: 슬라임, 46269: 스켈레톤, 46079: 슬라임, 46203: 스켈레톤, 46231: 스켈레톤, 46263: 스켈레톤, 46043: 슬라임, 46062: 슬라임, 46028: 슬라임, 46045: 슬라임, 46094: 슬라임, 46092: 슬라임, 46039: 슬라임, 46053: 슬라임, 46250: 스켈레톤, 46042: 슬라임, 46005: 슬라임, 46023: 슬라임, 46068: 슬라임, 46047: 슬라임, 46070: 슬라임, 46101: 슬라임, 46022: 슬라임, 46211: 스켈레톤, 46266: 스켈레톤, 46256: 스켈레톤, 46258: 스켈레톤, 46277: 스켈레톤, 46040: 슬라임, 46085: 슬라임, 46018: 슬라임, 46030: 슬라임, 46033: 슬라임, 46066: 슬라임, 46097: 슬라임, 46098: 슬라임, 46255: 스켈레톤, 46299: 스켈레톤, 46252: 스켈레톤, 46257: 스켈레톤, 46268: 스켈레톤, 46272: 스켈레톤, 46241: 스켈레톤, 46254: 스켈레톤, 46025: 슬라임, 46081: 슬라임, 46455: 구울, 46014: 슬라임, 46026: 슬라임, 46069: 슬라임, 46467: 구울, 46301: 스켈레톤, 46017: 슬라임, 46064: 슬라임, 46013: 슬라임, 46084: 슬라임, 46038: 슬라임, 46078: 슬라임, 46029: 슬라임, 46222: 스켈레톤, 46037: 슬라임, 46087: 슬라임, 46470: 구울, 46489: 구울, 46242: 스켈레톤, 46245: 스켈레톤, 46291: 스켈레톤, 46215: 스켈레톤, 46221: 스켈레톤, 46061: 슬라임, 46020: 슬라임, 46049: 슬라임, 46055: 슬라임, 46063: 슬라임, 46472: 구울, 46016: 슬라임, 46082: 슬라임, 46041: 슬라임, 46091: 슬라임, 46295: 스켈레톤, 46297: 스켈레톤, 46432: 구울, 46213: 스켈레톤, 46076: 슬라임, 46428: 구울, 46477: 구울, 46479: 구울, 46488: 구울, 46204: 스켈레톤, 46423: 구울, 46444: 구울, 46433: 구울, 46438: 구울, 46072: 슬라임, 46083: 슬라임, 46468: 구울, 46274: 스켈레톤, 46232: 스켈레톤, 46293: 스켈레톤, 46080: 슬라임, 46096: 슬라임, 46290: 스켈레톤, 46034: 슬라임, 46036: 슬라임, 46088: 슬라임, 46424: 구울, 46482: 구울, 46492: 구울, 46500: 구울, 46251: 스켈레톤, 46260: 스켈레톤, 46015: 슬라임, 46461: 구울, 46205: 스켈레톤, 46003: 슬라임, 46074: 슬라임, 46075: 슬라임, 46451: 구울, 46502: 구울, 46287: 스켈레톤, 46223: 스켈레톤, 46368: 머미, 46457: 구울, 46481: 구울, 46495: 구울, 46501: 구울, 46296: 스켈레톤, 46298: 스켈레톤, 46405: 구울, 46491: 구울, 46453: 구울, 46403: 구울, 46464: 구울, 46358: 머미, 46285: 스켈레톤, 46439: 구울, 46224: 스켈레톤, 46450: 구울, 46239: 스켈레톤, 46264: 스켈레톤, 46465: 구울, 46437: 구울, 46165: 좀비, 46427: 구울, 46430: 구울, 46463: 구울, 46414: 구울, 46431: 구울, 46436: 구울, 46339: 머미, 46352: 머미, 46434: 구울, 46325: 머미, 46407: 구울, 46417: 구울, 46493: 구울, 46333: 머미, 46366: 머미, 46443: 구울, 46126: 좀비, 46413: 구울, 46469: 구울, 46323: 머미, 46118: 좀비, 46186: 좀비, 46155: 좀비, 46170: 좀비, 46136: 좀비, 46362: 머미, 46360: 머미, 46130: 좀비, 46104: 좀비, 46151: 좀비, 46173: 좀비, 46189: 좀비, 46498: 구울, 46202: 좀비, 46127: 좀비, 46140: 좀비, 46121: 좀비, 46462: 구울, 46107: 좀비, 46184: 좀비, 46111: 좀비, 46378: 머미, 46381: 머미, 46377: 머미, 46163: 좀비, 46182: 좀비, 46156: 좀비, 46194: 좀비, 46356: 머미, 46148: 좀비, 46183: 좀비, 46185: 좀비, 46138: 좀비, 46134: 좀비, 46122: 좀비, 46198: 좀비, 46115: 좀비, 46171: 좀비, 46123: 좀비, 46157: 좀비, 46197: 좀비, 46188: 좀비, 46116: 좀비, 46149: 좀비, 46169: 좀비, 46375: 머미, 46153: 좀비, 46178: 좀비, 46166: 좀비, 46133: 좀비, 46110: 좀비, 46137: 좀비, 46176: 좀비, 46125: 좀비, 46147: 좀비, 46192: 좀비, 46174: 좀비, 46180: 좀비, 46114: 좀비, 46374: 머미, 46364: 머미, 46373: 머미, 46382: 머미, 46604: 버서커, 46658: 버서커, 46624: 버서커, 46603: 버서커, 46609: 버서커, 46665: 버서커, 46614: 버서커, 46613: 버서커, 46635: 버서커, 46644: 버서커, 46628: 버서커, 46641: 버서커, 46655: 버서커, 46652: 버서커, 46617: 버서커, 46640: 버서커, 46676: 버서커, 46610: 버서커, 46648: 버서커, 46667: 버서커, 46668: 버서커, 46637: 버서커, 46616: 버서커, 46618: 버서커, 46633: 버서커, 46612: 버서커, 46660: 버서커, 46615: 버서커, 46643: 버서커, 46680: 버서커, 46702: 버서커, 46611: 버서커, 46631: 버서커, 46619: 버서커, 46621: 버서커, 46632: 버서커, 46642: 버서커, 46649: 버서커, 46625: 버서커, 46607: 버서커, 46629: 버서커, 46630: 버서커, 46639: 버서커, 46606: 버서커, 46666: 버서커, 46663: 버서커, 46664: 버서커, 46645: 버서커, 46657: 버서커, 46673: 버서커, 46685: 버서커, 46692: 버서커, 46647: 버서커, 46696: 버서커, 46620: 버서커, 46651: 버서커, 46675: 버서커, 46677: 버서커, 46659: 버서커, 46670: 버서커, 46653: 버서커, 46686: 버서커, 46694: 버서커, 46695: 버서커, 46626: 버서커, 46654: 버서커, 46656: 버서커, 46671: 버서커, 46678: 버서커, 46661: 버서커, 46623: 버서커, 46605: 버서커, 46627: 버서커, 46662: 버서커, 46672: 버서커, 46688: 버서커, 46693: 버서커, 46674: 버서커, 46638: 버서커, 46681: 버서커, 46683: 버서커, 46687: 버서커, 46622: 버서커, 46646: 버서커, 46650: 버서커, 46682: 버서커, 46608: 버서커, 46679: 버서커, 46701: 버서커, 46684: 버서커, 46697: 버서커, 46690: 버서커, 46700: 버서커, 46636: 버서커, 46691: 버서커, 46689: 버서커, 46698: 버서커, 46699: 버서커, 46669: 버서커, 1893: 던전, 2098: 북, 2133: 동, 2135: 남, 53776: +6롱소드, 96969: 파란포션, 96972: 빨간포션, 97015: 스케일부츠, 97018: 무기강화주문서, 97038: 스케일팬츠, 47745: +5대거, 96968: 파란포션, 96992: 빨간포션, 47746: +5대거, 96967: 파란포션, 96981: 빨간포션, 83696: +6롱소드, 96971: 빨간포션, 96982: 파란포션, 97023: 방어구강화주문서, 97030: 스케일팬츠, 47739: +4레더팬츠, 53509: +4레더아머, 59094: +6롱소드, 72406: +4레더부츠, 72423: +4스케일쉴드, 79632: 림보귀환주문서, 86529: +4스케일벨트, 93332: 방어구강화주문서, 93684: 축복받은무기강화주문서, 96730: 무기강화주문서, 96974: 빨간포션, 96976: 파란포션, 97006: 저주받은방어구강화주문서, 1904: 던전, 2140: 북, 2172: 서, 2175: 동, 2177: 남, 1905: 던전, 2144: 북, 2176: 서, 2179: 동, 2181: 남, 282: 북, 314: 서, 317: 동, 319: 남, 49: 던전, 58: 던전, 60: 던전, 69: 던전, 1911: 던전, 2168: 북, 2200: 서, 2203: 동, 2205: 남, 1912: 던전, 2170: 북, 2204: 서, 2207: 남, 1918: 던전, 2194: 북, 2226: 서, 2229: 동, 2231: 남, 1914: 던전, 2178: 북, 2210: 서, 2213: 동, 2215: 남, 430: 북, 454: 서, 455: 동, 88: 던전, 97: 던전, 99: 던전, 764: 북, 796: 서, 799: 동, 801: 남, 515: 던전, 524: 던전, 526: 던전, 535: 던전, 1924: 던전, 2216: 북, 2248: 서, 2251: 동, 2253: 남, 97007: 파란포션, 97008: 빨간포션, 1926: 던전, 2224: 북, 2256: 서, 2259: 동, 2261: 남, 96993: 빨간포션, 97005: 파란포션, 97011: 파란포션, 97013: 빨간포션, 97027: 레더부츠, 97031: 나이프, 97033: 레더벨트, 97001: 빨간포션, 97009: 파란포션, 1928: 던전, 2232: 북, 2264: 서, 2267: 동, 2269: 남, 1034: 북, 1066: 서, 1069: 동, 1071: 남, 925: 던전, 934: 던전, 936: 던전, 945: 던전, 1929: 던전, 2236: 북, 2268: 서, 2271: 동, 2273: 남, 1930: 던전, 2240: 북, 2272: 서, 2275: 동, 2277: 남, 1931: 던전, 2244: 북, 2276: 서, 2279: 동, 2281: 남, 244: 북, 276: 서, 279: 동, 281: 남, 39: 던전, 48: 던전, 50: 던전, 59: 던전, 802: 북, 834: 서, 837: 동, 839: 남, 525: 던전, 534: 던전, 536: 던전, 545: 던전, 1941: 던전, 2282: 북, 2300: 서, 2301: 동, 426: 북, 452: 서, 453: 동, 87: 던전, 96: 던전, 98: 던전, 1936: 던전, 2262: 북, 2290: 서, 2291: 동, 1072: 북, 1104: 서, 1107: 동, 1109: 남, 935: 던전, 944: 던전, 946: 던전, 955: 던전, 1883: 던전, 2060: 북, 2095: 동, 2097: 남, 1843: 지하5층, 1943: 동, 1945: 남, 47727: 위, 47728: 아래, 47713: 파란포션, 240: 북, 272: 서, 275: 동, 277: 남, 38: 던전, 47: 던전, 1849: 던전, 1964: 서, 1967: 동, 1969: 남, 236: 북, 268: 서, 271: 동, 273: 남, 37: 던전, 46: 던전, 57: 던전, 798: 북, 830: 서, 833: 동, 835: 남, 533: 던전, 544: 던전, 1859: 던전, 1970: 북, 2002: 서, 2005: 동, 2007: 남, 232: 북, 264: 서, 267: 동, 269: 남, 36: 던전, 45: 던전, 56: 던전, 794: 북, 829: 동, 831: 남, 523: 던전, 543: 던전, 1854: 던전, 1950: 북, 1982: 서, 1985: 동, 1987: 남, 1148: 북, 1180: 서, 1183: 동, 1185: 남, 964: 던전, 966: 던전, 975: 던전, 1110: 북, 1142: 서, 1145: 동, 1147: 남, 954: 던전, 956: 던전, 965: 던전, 1863: 던전, 1984: 북, 2019: 동, 2021: 남, 1867: 던전, 2000: 북, 2032: 서, 2035: 동, 2037: 남, 1880: 던전, 2050: 북, 2082: 서, 2085: 동, 2087: 남, 756: 북, 791: 동, 793: 남, 513: 던전, 1873: 던전, 2022: 북, 2057: 동, 2059: 남, 1890: 던전, 2088: 북, 2120: 서, 2123: 동, 2125: 남, 1892: 던전, 2094: 북, 2128: 서, 2131: 남, 1114: 북, 1146: 서, 1149: 동, 1151: 남, 957: 던전, 1118: 북, 1150: 서, 1153: 동, 1155: 남, 947: 던전, 958: 던전, 967: 던전, 1896: 던전, 2110: 북, 2142: 서, 2145: 동, 2147: 남, 1894: 던전, 2102: 북, 2134: 서, 2137: 동, 2139: 남, 1080: 북, 1112: 서, 1115: 동, 1117: 남, 937: 던전, 948: 던전, 718: 북, 753: 동, 755: 남, 503: 던전, 514: 던전, 1906: 던전, 2148: 북, 2180: 서, 2183: 동, 2185: 남, 1921: 던전, 2206: 북, 2238: 서, 2241: 동, 2243: 남, 194: 북, 226: 서, 229: 동, 231: 남, 26: 던전, 35: 던전, 1042: 북, 1074: 서, 1077: 동, 1079: 남, 927: 던전, 938: 던전, 434: 북, 456: 서, 457: 동, 89: 던전, 100: 던전, 1038: 북, 1070: 서, 1073: 동, 1075: 남, 926: 던전, 1927: 던전, 2228: 북, 2260: 서, 2263: 동, 2265: 남, 1046: 북, 1078: 서, 1081: 동, 1083: 남, 928: 던전, 939: 던전, 1938: 던전, 2270: 북, 2294: 서, 2295: 동, 1935: 던전, 2258: 북, 2288: 서, 2289: 동, 438: 북, 458: 서, 459: 동, 90: 던전, 101: 던전, 1076: 북, 1108: 서, 1111: 동, 1113: 남, 1845: 던전, 1948: 서, 1951: 동, 1953: 남, 1850: 던전, 1968: 서, 1971: 동, 1973: 남, 1862: 던전, 1980: 북, 2014: 서, 2017: 남, 1866: 던전, 1996: 북, 2028: 서, 2031: 동, 2033: 남, 1878: 던전, 2042: 북, 2074: 서, 2077: 동, 2079: 남, 1874: 던전, 2026: 북, 2058: 서, 2061: 동, 2063: 남, 1885: 던전, 2068: 북, 2100: 서, 2103: 동, 2105: 남, 1040: 서, 1043: 동, 1045: 남, 929: 던전, 760: 북, 792: 서, 795: 동, 797: 남, 156: 북, 188: 서, 191: 동, 193: 남, 16: 던전, 25: 던전, 27: 던전, 1044: 서, 1047: 동, 1049: 남, 930: 던전, 1897: 던전, 2114: 북, 2146: 서, 2149: 동, 2151: 남, 1895: 던전, 2106: 북, 2138: 서, 2141: 동, 2143: 남, 1903: 던전, 2136: 북, 2171: 동, 2173: 남, 1048: 서, 1051: 동, 1053: 남, 931: 던전, 940: 던전, 67745: 운영실, 53552: +5액스, 61046: 매직스태프, 1920: 던전, 2202: 북, 2234: 서, 2237: 동, 2239: 남, 768: 북, 800: 서, 803: 동, 805: 남, 516: 던전, 527: 던전, 1923: 던전, 2212: 북, 2247: 동, 2249: 남, 1052: 서, 1055: 동, 1057: 남, 932: 던전, 941: 던전, 696: 북, 728: 서, 731: 동, 733: 남, 497: 던전, 506: 던전, 508: 던전, 517: 던전, 2: 림보, 47704: 동, 54929: 루천, 54930: 루아, 54946: 루카, 56108: 루루, 56113: 루리, 57317: Wanner, 60998: qunosores, 62691: dongmasu, 62831: musemu, 63657: 루나, 69217: test, 69218: 모모, 76052: 군감, 76123: 정현, 77556: 카렐, 96950: 다크버드, 97017: 게시판, 47703: 광장, 97039: 가을, 806: 북, 838: 서, 841: 동, 843: 남, 537: 던전, 546: 던전, 103: 동, 107: 동, 111: 동, 115: 동, 119: 동, 123: 동, 127: 동, 131: 동, 135: 동, 141: 동, 145: 동, 149: 동, 153: 동, 157: 동, 161: 동, 165: 동, 169: 동, 173: 동, 179: 동, 183: 동, 187: 동, 195: 동, 199: 동, 203: 동, 207: 동, 211: 동, 217: 동, 221: 동, 225: 동, 233: 동, 237: 동, 241: 동, 245: 동, 249: 동, 255: 동, 259: 동, 263: 동, 283: 동, 287: 동, 293: 동, 297: 동, 301: 동, 305: 동, 309: 동, 313: 동, 321: 동, 325: 동, 331: 동, 335: 동, 339: 동, 343: 동, 347: 동, 351: 동, 355: 동, 359: 동, 363: 동, 369: 동, 373: 동, 377: 동, 381: 동, 385: 동, 389: 동, 393: 동, 397: 동, 401: 동, 407: 동, 411: 동, 415: 동, 419: 동, 423: 동, 427: 동, 431: 동, 435: 동, 439: 동, 445: 동, 447: 동, 449: 동, 451: 동, 461: 동, 563: 동, 567: 동, 571: 동, 575: 동, 579: 동, 583: 동, 587: 동, 591: 동, 595: 동, 601: 동, 605: 동, 609: 동, 613: 동, 617: 동, 621: 동, 625: 동, 629: 동, 633: 동, 639: 동, 643: 동, 647: 동, 651: 동, 655: 동, 659: 동, 663: 동, 667: 동, 671: 동, 677: 동, 681: 동, 685: 동, 689: 동, 693: 동, 697: 동, 701: 동, 705: 동, 709: 동, 715: 동, 719: 동, 723: 동, 727: 동, 735: 동, 739: 동, 743: 동, 747: 동, 757: 동, 761: 동, 765: 동, 769: 동, 773: 동, 777: 동, 781: 동, 785: 동, 807: 동, 811: 동, 815: 동, 819: 동, 823: 동, 845: 동, 849: 동, 853: 동, 857: 동, 861: 동, 867: 동, 871: 동, 875: 동, 879: 동, 883: 동, 887: 동, 891: 동, 895: 동, 899: 동, 905: 동, 907: 동, 909: 동, 911: 동, 913: 동, 915: 동, 917: 동, 919: 동, 921: 동, 1023: 동, 1027: 동, 1031: 동, 1035: 동, 1039: 동, 1061: 동, 1065: 동, 1085: 동, 1089: 동, 1093: 동, 1099: 동, 1103: 동, 1119: 동, 1123: 동, 1127: 동, 1131: 동, 1137: 동, 1141: 동, 1157: 동, 1161: 동, 1165: 동, 1169: 동, 1175: 동, 1179: 동, 1187: 동, 1191: 동, 1195: 동, 1199: 동, 1203: 동, 1207: 동, 1213: 동, 1217: 동, 1221: 동, 1225: 동, 1229: 동, 1233: 동, 1237: 동, 1241: 동, 1245: 동, 1251: 동, 1255: 동, 1259: 동, 1263: 동, 1267: 동, 1271: 동, 1275: 동, 1279: 동, 1283: 동, 1289: 동, 1293: 동, 1297: 동, 1301: 동, 1305: 동, 1309: 동, 1313: 동, 1317: 동, 1321: 동, 1327: 동, 1331: 동, 1335: 동, 1339: 동, 1343: 동, 1347: 동, 1351: 동, 1355: 동, 1359: 동, 1365: 동, 1367: 동, 1369: 동, 1371: 동, 1373: 동, 1375: 동, 1377: 동, 1379: 동, 1381: 동, 1483: 동, 1487: 동, 1491: 동, 1495: 동, 1499: 동, 1503: 동, 1507: 동, 1511: 동, 1515: 동, 1521: 동, 1525: 동, 1529: 동, 1533: 동, 1537: 동, 1541: 동, 1545: 동, 1549: 동, 1553: 동, 1559: 동, 1563: 동, 1567: 동, 1571: 동, 1575: 동, 1579: 동, 1583: 동, 1587: 동, 1591: 동, 1597: 동, 1601: 동, 1605: 동, 1609: 동, 1613: 동, 1617: 동, 1621: 동, 1625: 동, 1629: 동, 1635: 동, 1639: 동, 1643: 동, 1647: 동, 1651: 동, 1655: 동, 1659: 동, 1663: 동, 1667: 동, 1673: 동, 1677: 동, 1681: 동, 1685: 동, 1689: 동, 1693: 동, 1697: 동, 1701: 동, 1705: 동, 1711: 동, 1715: 동, 1719: 동, 1723: 동, 1727: 동, 1731: 동, 1735: 동, 1739: 동, 1743: 동, 1749: 동, 1753: 동, 1757: 동, 1761: 동, 1765: 동, 1769: 동, 1773: 동, 1777: 동, 1781: 동, 1787: 동, 1791: 동, 1795: 동, 1799: 동, 1803: 동, 1807: 동, 1811: 동, 1815: 동, 1819: 동}

**3. @py self.location.contents_cache.clear()**

**4. @py self.location.contents_cache._idcache**
  Same result as 2. (There are a lot of contents of 2, and some were deleted.)

**Comment by aceamro** [comment]
@Griatch
The accumulation of the character's _idcache every time it moves makes sense for faster retrieval.
But, Each time you move, the location _idcache is also accumulated. Is it correct?

**Comment by aceamro** [comment]
@Griatch
I don't know if this is right
Adding the **self._idcache = self.obj .__ class __.__ instance_cache__** part to the "class ContentsHandler(object) >> def init(self):".
Nevertheless, we will watch for problems.
 
evennia/objects/models.py line 54
    def init(self):
        """
        Re-initialize the content cache

        """
        self._idcache = self.obj.__class__.__instance_cache__
        self._pkcache.update(
            dict((obj.pk, None) for obj in ObjectDB.objects.filter(db_location=self.obj) if obj.pk)
        )

**Comment by strikaco** [comment]
> In #1064, the reason for this error was the firing of the cache flush mechanism (see my comment that reproduced it). This will start firing if the memory footprint of your game is increasing past the memory limiter breakoff point as set in settings. Is the memory limiter firing for you in the same way now?

@Griatch I'm starting to experience this error myself. The memory limiter is definitely kicking in; the rest of the conditions are as reported-- 12h+ uptime, active (10k+ objects created).

But everything still *seems* to be working inasmuch as I can observe it. Is this log entry misclassified (should it be a `warn` event?) or is its presence indicative of an actual error that needs addressing?

**Comment by Griatch** [comment]
@TehomCD, @aceamro @strikaco I have investigated the error and found this to be benign. What happens is that the central idmapper cache is flushed (this happens on a timer to cut down on memory usage), meaning the lookup fails. Reloading from database rebuilds that part of the cache and it will after that not longer give the error. So this is the normal operation of rebuilding the cache after a flush. It's just the 'error' in the log that's confusing. 
I have removed the logging of this (it shouldn't even be a warning).
